crates/biome_css_parser/tests/css_test_suite/ok/scss/value/signed-numeric-operator.scss

```
@@ -28,4 +28,8 @@ foo {
     (1 +2): foo,
     ($gap +1): bar,
   );
+
+  dimension-minus-left-space: 10 -10px;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)